src/Hng.Application.Test/Hng.Application.Test.csproj

```diff
@@ -1,22 +1,28 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>net8.0</TargetFramework>
-    <ImplicitUsings>enable</ImplicitUsings>
-    <Nullable>enable</Nullable>
-  </PropertyGroup>
+	<PropertyGroup>
+		<TargetFramework>net8.0</TargetFramework>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+	</PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="coverlet.collector" Version="6.0.2">

```

Comment:
Why did you remove `Xunit`?

Reply:
I needed a framework suitable for projects that rely heavily on data-driven tests. To be able to use [SetUp], [TearDown], [OneTimeSetUp], and [OneTimeTearDown] attributes for setup and teardown logic at the test and fixture levels rather than using constructors or IDisposable.